primitives/core/src/ecdsa.rs

```diff
@@ -95,6 +95,7 @@ impl Public {
 }
 
 impl TraitPublic for Public {
+	const KEY_KIND_ID: &'static str = "sr25519";
```

Comment:
Why this is inside a trait directly and in ed25519 is defined through some const `ED25519_KIND_ID`?